### ISSUE TITLE
Using the original wireless Buzzers crashes RPCS3

### ISSUE THREAD

**Issue by kapoios** [RPCS3/rpcs3#12367]
## Quick summary
Using the original wireless Buzzers crashes RPCS3 (tried BCES00647 and BCES00831)

## Details

While using the original wireless controllers I can navigate the menus etc but I get multiple:
"sys_usbd: Unexpected error from libusb_submit_transfer: -5"
If I wait 30 seconds to 1 minute the errors accumulate and RPCS3 locks up with:
"sys_usbd: Usb transfers are saturated!"

RPCS3's log.
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/9141164/RPCS3.zip)

Screenshot
![2022-07-19 13_36_06-RPCS3 0 0 23-13947-5e11b2b0 Alpha _ master](https://user-images.githubusercontent.com/10677913/179754602-5a11d05c-3332-4b6f-81e6-c47f7e45be7e.png)

System configuration:
Win10
i9-9900K
3090
516.59

**Comment by RipleyTom** [comment]
Did you follow these instructions: https://wiki.rpcs3.net/index.php?title=Help:Peripherals_and_accessories#On_Windows ?

**Comment by kapoios** [comment]
> Did you follow these instructions: https://wiki.rpcs3.net/index.php?title=Help:Peripherals_and_accessories#On_Windows ?

Yes, something seems was bugging with the winusb driver, I removed and added it again and is ok now, closing this.
